plugins/modules/zos_fetch.py

```diff
@@ -20,21 +20,18 @@
 version_added: "2.9"
 short_description: Fetch data from z/OS
 description:
-  - This module copies a file or data set from a remote z/OS system to the
-    local machine. Use the
-    L(fetch,https://docs.ansible.com/ansible/latest/modules/fetch_module.html)
-    module to copy files from local machine to the remote z/OS system.
+  - This module fetches a file or data set from a remote z/OS system. 
   - When fetching a sequential data set, the destination file name will be the
     same as the data set name.
-  - When fetching a PDS/PDS(E), the destination will be a directory with the
-    same name as the PDS/PDS(E).
+  - When fetching a PDS or PDSE, the destination will be a directory with the
+    same name as the PDS or PDSE.
   - Files that already exist at dest will be overwritten if they are different
```

Comment:
When fetching files that present on the destination I(dest), they will be overwritten if they are different than the source I(src).

Reply:
@asifmahmud - I rewrote the above a bit, but what is the behavior if the source and destination are the same, it might be further down in this doc but noting it now.

Reply:
@ddimatos Actually, the behavior is that `dest` will be overwritten regardless, consistent with Ansible's fetch behavior